beacon-chain/blockchain/service.go

```diff
@@ -271,14 +264,21 @@ func (c *ChainService) processBlock(message proto.Message) {
 
 // ApplyForkChoiceRule determines the current beacon chain head using LMD GHOST as a block-vote
 // weighted function to select a canonical head in Ethereum Serenity.
-func (c *ChainService) ApplyForkChoiceRule(block *pb.BeaconBlock, computedState *pb.BeaconState) error {
+func (c *ChainService) ApplyForkChoiceRule(block *pb.BeaconBlock, postState *pb.BeaconState) error {
```

Comment:
Since we think this might take a long time, can you add a `context.Context` and start a span for apply fork choice?